pkg/extensionController/registry/localDir.go

```diff
@@ -19,7 +19,8 @@ type localDirRegistry struct {
 	dir string
 }
 
-// FindExtensions searches for .js files in the given directory
+// FindExtensions searches for .js files in the local registry directory.
+// [impl -> dsn~extension-definitions-storage~1]
```

Comment:
Unfortunately godoc will include this tracing comment into user visible doc :(

Reply:
Good point. Replacing the // comments with /* */ solves this.